source/Calamari/Kubernetes/Commands/Executors/GatherAndApplyRawYamlExecutor.cs

```diff
@@ -50,6 +51,12 @@ protected override IEnumerable<ResourceIdentifier> ApplyAndGetResourceIdentifier
             foreach (var directory in directories)
             {
                 var res = ApplyBatchAndReturnResourceIdentifiers(directory).ToList();
```

Comment:
I think you can change `.ToList()` to `.ToArray()` here and remove the `.ToArray()` on line 57.

Reply:
Yes good catch!